.github/actions/require-multiple-reviewers-v1/README.md

```diff
@@ -0,0 +1,55 @@
+# require-multiple-reviewers-v1
+
+A GitHub Action that requires multiple reviewers for important files
+
+## Inputs
+
+| Name                   | Required | Description                                                | Default |
+| ---------------------- | -------- | ---------------------------------------------------------- | ------- |
+| `token`                | Yes      | A GitHub token with the required permissions               | NA      |
+| `number-of-reviewers`  | Yes      | The number of reviewers required.                          | 2       |
+| `changed-files-path`   | Yes      | The path to the file containing a list of changed files.   | NA      |
```

Comment:
Can this be more specific about the expected formats of the files? How is the list separated? What are the file paths relative to? Does it support any sort of globbing/patterns? (it's ok for this to start simple but it should be documented)